emsdk.py

```diff
@@ -2589,17 +2584,12 @@ def to_msys_path(p):
 
 # Looks at the current PATH and adds and removes entries so that the PATH reflects
 # the set of given active tools.
-def adjusted_path(tools_to_activate, log_additions=False, system_path_only=False):
+def adjusted_path(tools_to_activate, log_additions=False, system=False):
```

Comment:
Looks like the `log_additions` param is never uses, or passed.  Unrelated to this PR but since your are changing that line maybe delete it?